### PR TITLE
zh-translation: content/zh/docs/examples/microservices-istio/logs-istio

### PR DESCRIPTION
**Pull request by nicole-lihui**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/examples/microservices-istio/logs-istio/index.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+2.  在 _Expression_ 输入框中运行以下示例查询。按下 _Execute_ 按钮，在 _Console_ 中查看查询结果。这个查询使用 `tutorial` 作为应用的命名空间，你可以替换成你的命名空间。在查询数据时，为了能够得到更棒的效果，请运行前面步骤中描述的实时流量模拟器。
+
+    1. 查询你命名空间的所有请求：
+
+        {{< text plain >}}
+        istio_requests_total{destination_service_namespace="tutorial", reporter="destination"}
+        {{< /text >}}
+
+    2.  查询你的命名空间请求的总和：
```
Comment (kebe7jun) [inline content/zh/docs/examples/microservices-istio/logs-istio/index.md:30]:
```suggestion
    2.  查询命名空间请求的总和：
```

File: content/zh/docs/examples/microservices-istio/logs-istio/index.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+请看下面几个 Prometheus 查询 Istio-related 的例子。
+
+1.  通过 [http://my-istio-logs-database.io](http://my-istio-logs-database.io) 访问 Prometheus UI 界面。（这  `my-istio-logs-database.io` URL 在你[之前配置](/zh/docs/examples/microservices-istio/bookinfo-kubernetes/#update-your-etc-hosts-configuration-file)的 `/etc/hosts` 文件中）。
+
+    {{< image width="80%" link="prometheus.png" caption="Prometheus Query UI" >}}
+
+2.  在 _Expression_ 输入框中运行以下示例查询。按下 _Execute_ 按钮，在 _Console_ 中查看查询结果。这个查询使用 `tutorial` 作为应用的命名空间，你可以替换成你的命名空间。在查询数据时，为了能够得到更棒的效果，请运行前面步骤中描述的实时流量模拟器。
+
+    1. 查询你命名空间的所有请求：
```
Comment (kebe7jun) [inline content/zh/docs/examples/microservices-istio/logs-istio/index.md:24]:
```suggestion
    1. 查询命名空间的所有请求：
```

File: content/zh/docs/examples/microservices-istio/logs-istio/index.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        sum(istio_requests_total{destination_service_namespace="tutorial", reporter="destination"})
+        {{< /text >}}
+
+    3.  查询 `reviews` 微服务的请求：
+
+        {{< text plain >}}
+        istio_requests_total{destination_service_namespace="tutorial", reporter="destination",destination_service_name="reviews"}
+        {{< /text >}}
+
+    4.  在过去5分钟内， `reviews` 微服务实例中的所有请求的[请求率](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate)：
```
Comment (kebe7jun) [inline content/zh/docs/examples/microservices-istio/logs-istio/index.md:42]:
```suggestion
    4.  在过去5分钟内， `reviews` 微服务实例中的所有请求的[请求速率](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate)：
```

File: content/zh/docs/examples/microservices-istio/logs-istio/index.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+通过这些 `demo` 安装任务是初学者进一步了解 Istio 的方式：
+
+- [请求路由](/zh/docs/tasks/traffic-management/request-routing/)
+- [失败注入](/zh/docs/tasks/traffic-management/fault-injection/)
+- [流量转移](/zh/docs/tasks/traffic-management/traffic-shifting/)
+- [查询指标](/zh/docs/tasks/observability/metrics/querying-metrics/)
+- [可视化指标](/zh/docs/tasks/observability/metrics/using-istio-dashboard/)
+- [访问外部服务](/zh/docs/tasks/traffic-management/egress/egress-control/)
+- [可视化您的网格](/zh/docs/tasks/observability/kiali/)
```
Comment (kebe7jun) [inline content/zh/docs/examples/microservices-istio/logs-istio/index.md:62]: https://istio.io/latest/zh/docs/tasks/observability/kiali/

这个 title 是 网络可视化。
```suggestion
- [网络可视化](/zh/docs/tasks/observability/kiali/)
```

File: content/zh/docs/examples/microservices-istio/logs-istio/index.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+上面使用的请求采用 `istio_requests_total` 指标，这是一个标准的 Istio 指标。你可以观察其他指标，特别是 Envoy （[Envoy](https://www.envoyproxy.io) 是 Istio 的边车代理）。你可以在 _insert metric at cursor_ 下拉菜单的看到收集的数据记录。
+
+## 下一步{#next-steps}
+
+祝贺完成本教程！
+
+通过这些 `demo` 安装任务是初学者进一步了解 Istio 的方式：
+
+- [请求路由](/zh/docs/tasks/traffic-management/request-routing/)
+- [失败注入](/zh/docs/tasks/traffic-management/fault-injection/)
```
Comment (kebe7jun) [inline content/zh/docs/examples/microservices-istio/logs-istio/index.md:57]: 这几个描述都去看一下实际文档的 title。

**Review by kebe7jun — APPROVED** [review]

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**